packages/Takeover/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "@paprika/takeover",
+  "version": "0.1.0",
+  "description": "Takeover component",
+  "author": "@paprika",
+  "main": "lib/index.js",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/acl-services/paprika/tree/master/packages/Takeover"
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "dependencies": {
+    "@babel/runtime-corejs2": "^7.3.1",
+    "@paprika/button": "^0.1.13",
```

Comment:
we recently upgrade paprika with the breaking change on the L10n component issue.

you need to bump button to 0.2.0 at least and I'm not sure the other ones but you can check their own package json to see the latest version.